### PR TITLE
fix: Alert if publisher doesn't have cargo-release installed

### PR DESCRIPTION
**Pull request by CPerezz**
Resolves: #61 (or at least prevents it to happen again)

### CODE REVIEW

**Review by huitseeker — CHANGES_REQUESTED** [review]
For me, `cargo-release --version` fails on a machine with `cargo-release` installed, see details in inline message.

File: Makefile [hunk 1]
```diff
@@ -8,6 +8,9 @@ doc:
 release:
 ifndef VERSION
 	$(error VERSION is not set. Run with `make VERSION=<version> release`)
+endif
+ifeq (, $(shell cargo-release --version))
```
[the remark this answers is not in the document]
Comment (CPerezz) [reply Makefile:12]: I should have tried with the thing installed LOL!
Let me try with grep yes! Thanks for the suggestion @huitseeker !

**Comment by CPerezz** [comment]
@huitseeker thanks a lot for the suggestion. I should have tested the positive case before filing this. 
Now it should work with your correct suggestion (just hoping there aren't many `release` keywords for other widely used cargo sub-commands hehehe.

**Review by huitseeker — APPROVED** [review]
🙏

**Review by han0110 — APPROVED** [review]
LGTM

**Pull request merged**